packages/react-scripts/template/src/App.css

```diff
@@ -1,21 +1,28 @@
 .App {
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: center;
+  justify-content: center;
+  min-height: 100vh;
+  padding: 3rem;
   text-align: center;
 }
 
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 80px;
+.App > *+* {
```

Comment:
I would like to avoid selectors like this. We are generally trying to encourage people to "think in components", and any more fancy selectors often break when you nest one component into another.